<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CArmHardwareControl</class>
 <widget class="QWidget" name="CArmHardwareControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>328</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CArmHardwareControl</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel{
font-size:14px;
font-family:NotoSansCJK-Regular,NotoSansCJK;
font-weight:400;
color:rgba(177,193,214,1);
line-height:21px;
}

QPushButton{
background:rgba(0,0,0,0);
opacity:0.7;
}</string>
  </property>
  <widget class="QLabel" name="left_rightLbe">
   <property name="geometry">
    <rect>
     <x>161</x>
     <y>7</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>-/-</string>
   </property>
  </widget>
  <widget class="QLabel" name="angleLbe">
   <property name="geometry">
    <rect>
     <x>278</x>
     <y>7</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>-/-</string>
   </property>
  </widget>
  <widget class="QLabel" name="up_downLbe">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>7</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>-/-</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>127</x>
     <y>9</y>
     <width>20</width>
     <height>11</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
image:url(:/control/Resource/control/front_back.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>244</x>
     <y>7</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
image:url(:/control/Resource/control/rotate.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>27</x>
     <y>36</y>
     <width>271</width>
     <height>261</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
image:url(:/control/Resource/control/robot_arm.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>23</x>
     <y>4</y>
     <width>11</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
image:url(:/control/Resource/control/up_down.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="anticlockwiseBtn">
   <property name="geometry">
    <rect>
     <x>61</x>
     <y>42</y>
     <width>90</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
image:url(:/control/Resource/control/anticlockwise.png)
}
QPushButton:pressed{
image:url(:/control/Resource/control/anticlockwise_press.png)
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="limiterHwideBtn">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>286</y>
     <width>64</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
image:url(:/control/Resource/control/limiter_Hwide.png)
}
QPushButton:pressed{
image:url(:/control/Resource/control/limiter_Hwide_press.png)
}
QPushButton:checked{
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="limiterHnarrowBtn">
   <property name="geometry">
    <rect>
     <x>94</x>
     <y>286</y>
     <width>64</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
image:url(:/control/Resource/control/limiter_Hnarrow.png)
}
QPushButton:pressed{
image:url(:/control/Resource/control/limiter_Hnarrow_press.png)
}
QPushButton:checked{
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="limiterVwideBtn">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>286</y>
     <width>64</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
image:url(:/control/Resource/control/limiter_Vwide.png)
}
QPushButton:pressed{
image:url(:/control/Resource/control/limiter_Vwide_press.png)
}
QPushButton:checked{
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="limiterVnarrowBtn">
   <property name="geometry">
    <rect>
     <x>246</x>
     <y>286</y>
     <width>64</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
image:url(:/control/Resource/control/limiter_Vnarrow.png)
}
QPushButton:pressed{
image:url(:/control/Resource/control/limiter_Vnarrow_press.png)
}
QPushButton:checked{
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="frontBtn">
   <property name="geometry">
    <rect>
     <x>39</x>
     <y>139</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
image:url(:/control/Resource/control/front.png)
}
QPushButton:pressed{
image:url(:/control/Resource/control/front_press.png)
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="backBtn">
   <property name="geometry">
    <rect>
     <x>99</x>
     <y>139</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
image:url(:/control/Resource/control/back.png)
}
QPushButton:pressed{
image:url(:/control/Resource/control/back_press.png)
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="upBtn">
   <property name="geometry">
    <rect>
     <x>196</x>
     <y>100</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
image:url(:/control/Resource/control/up.png)
}
QPushButton:pressed{
image:url(:/control/Resource/control/up_press.png)
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="downBtn">
   <property name="geometry">
    <rect>
     <x>196</x>
     <y>178</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
image:url(:/control/Resource/control/down.png)
}
QPushButton:pressed{
image:url(:/control/Resource/control/down_press.png)
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="clockwiseBtn">
   <property name="geometry">
    <rect>
     <x>61</x>
     <y>216</y>
     <width>90</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
image:url(:/control/Resource/control/clockwise.png)
}
QPushButton:pressed{
image:url(:/control/Resource/control/clockwise_press.png)
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="lampBtn">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>30</y>
     <width>42</width>
     <height>42</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
image:url(:/control/Resource/control/lamp_off.png)
}
QPushButton:checked{
image:url(:/control/Resource/control/lamp_on.png)
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="warnBtn">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>77</y>
     <width>42</width>
     <height>42</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
image:url(:/control/Resource/control/warning_off.png)
}
QPushButton:checked{
image:url(:/control/Resource/control/warning_on.png)
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="CArmTouchPanel.qrc"/>
 </resources>
 <connections/>
</ui>
